ui/component/common/file-selector.jsx

```diff
@@ -47,7 +47,24 @@ class FileSelector extends React.PureComponent<Props> {
   };
 
   handleDirectoryInputSelection = () => {
-    remote.dialog.showOpenDialog({ properties: ['openDirectory'] }).then((result) => {
+    var defaultPath;
+    var properties;
+    var isWin = process.platform === 'win32';
+    var type = this.props.type;
+
+    if (isWin === true) {
+      defaultPath = process.env.HOME || process.env.HOMEPATH || process.env.USERPROFILE;
+    }
+
+    if (type === 'openFile') {
+      properties = ['openFile'];
+    }
+
+    if (type === 'openDirectory') {
+      properties = ['openDirectory'];
+    }
+
+    remote.dialog.showOpenDialog({ properties, defaultPath }).then((result) => {
```

Comment:
nice, 
tested in console in linux
`x = require('@electron/remote')`
`x.dialog.showOpenDialog({properties: ['openFile'], defaultPath: undefined})`
And it worked fine.